src/map/utils/synthutils.cpp

```diff
@@ -290,7 +290,7 @@ namespace synthutils
                 success = std::clamp(success, 0.05, 0.95);
 
                 // Apply synthesis success rate modifier
-                modSynthSuccess = PChar->CraftContainer->getCraftType() == CRAFT_SYNTHESIS ? PChar->getMod(Mod::SYNTH_SUCCESS) : PChar->getMod(Mod::DESYNTH_SUCCESS);
+                modSynthSuccess = PChar->CraftContainer->getCraftType() == CRAFT_DESYNTHESIS ? PChar->getMod(Mod::DESYNTH_SUCCESS) : PChar->getMod(Mod::SYNTH_SUCCESS);
```

Comment:
Isnt this doing the same thing it already was?

Reply:
Not quite because only one craft type for desynthesis, while three craft types for synthesis (which all need to have the Mod::SYNTH_SUCCESS applied). Thus without this change the other types of synthesis would get Mod::DESYNTH_SUCCESS.